### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Andre3567**
- Made compatible with Android 14
- Updated dependencies 
- Migrated from Exoplayer2 to Media3

### CODE REVIEW

File: player/src/main/java/com/tritondigital/player/MediaPlayer.java [hunk 1]
```diff
@@ -684,12 +660,7 @@ final void setState(int state) {
         }
     }
 
-    final void notifyAnalytics(Format format) {
```
Comment (Andrei-C96) [inline player/src/main/java/com/tritondigital/player/MediaPlayer.java:687]: @Andre3567 Are some parts of the analytics being removed?

**Review by Andrei-C96 — APPROVED** [review]
LGTM!

**Review by dral3x — APPROVED** [review]
Looks ok.

File: player/src/main/java/com/tritondigital/player/MediaPlayer.java (hunk 1 above)
Comment (Andre3567) [reply player/src/main/java/com/tritondigital/player/MediaPlayer.java:687]: Yes, there isn't supposed to be any analytics in the public repo

**Pull request merged**